gladier_xpcs/flows/flow_gpu.py

```diff
@@ -1,20 +1,29 @@
-from gladier import generate_flow_definition
+"""
+XPCS Online GPU Flow
+
+Summary: This flow executes new xpcs_boost GPU flow.
+- Data is transfered from Clutch to Theta
+- An "empty" publication is added to the index
+- Nodes are "warmed"
+- Eigen-Corr is applied
+- Plots are made
+- Gather + Publish the final data to the portal
+"""
+from gladier import GladierBaseClient, generate_flow_definition, utils
+from gladier_xpcs.flows.container_flow_base import ContainerBaseClient
+
 # import gladier_xpcs.log  # Uncomment for debug logging
-from gladier_xpcs.flow_base import XPCSBaseClient
 
 
 @generate_flow_definition(modifiers={
    'publish_gather_metadata': {'payload': '$.GatherXpcsMetadata.details.result[0]'}
 })
-class XPCSGPUFlow(XPCSBaseClient):
-    # See flow_base.py for assigning containers.
-    containers = {}
-
+class XPCSGPUFlow(GladierBaseClient):
     gladier_tools = [
         'gladier_xpcs.tools.TransferFromClutchToTheta',
         'gladier_xpcs.tools.PrePublish',
         'gladier_xpcs.tools.AcquireNodes',
-        'gladier_xpcs.tools.EigenCorrGPU',
+        'gladier_xpcs.tools.XPCSBoost',
```

Comment:
Maybe BoostCorr would be a better name to match the tool?

Reply:
Yeah. Still leaving the function inside as xpcs_boost_corr so it does not clash with the import boost_corr on the rest of the code.

Reply:
Sounds good